src/bp/ui-studio/src/web/components/Shared/ElementsList/ContentElement.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
```

Comment:
Is it possible to use ContentPickerWidget instead ? We have lot of duplicate code

Reply:
@allardy I would need to make some changes on it, is that ok to you?